deployments/docker/src/builder.docker

```diff
@@ -57,7 +57,7 @@ RUN go mod download &&\
     go install google.golang.org/protobuf/cmd/protoc-gen-go@v1.26 &&\
     go install google.golang.org/grpc/cmd/protoc-gen-go-grpc@v1.1 &&\
     go install github.com/mitchellh/protoc-gen-go-json@v1.1.0 &&\
-    go install github.com/veraison/corim/cocli@latest &&\
+    go install github.com/veraison/corim/cocli@eeb7bd48 &&\
```

Comment:
I thought we strongly enforce revisions now?

Reply:
That hash is assocated with a v2.0.0rc0 tag, however because it's a v2 tag for a modules without a `/v2`, using it won't work. (conversely, moving to `/v2` without publishing a release won't work either), it's a catch22 in the go module versioning system, it seems. So  have to use the hash here.